music-player/src/components/RemoveSongFromPlaylist.tsx

```diff
@@ -0,0 +1,16 @@
+import { Modal } from 'antd';
+import { Dispatch } from 'redux';
+import { getRemoveSongFromPlaylistAction } from '../actions';
+
+export function showDeleteConfirm(dispatch: Dispatch, songIndex: number) {
```

Comment:
This doesn't deserve its own file.
Also, receive the callback and not dispatch!